include/rocksdb/env.h

```diff
@@ -1148,6 +1148,8 @@ class Directory {
   virtual ~Directory() {}
   // Fsync directory. Can be called concurrently from multiple threads.
   virtual Status Fsync() = 0;
+  // Close directory.
+  virtual Status Close() = 0;
```

Comment:
This is not a backward compatible change. We should provide a default implementation rather than defining this to be pure virtual. RocksDB has a policy of not making breaking changes in minor releases.

Reply:
@littlepig2013 Yeah I overlooked this pure virtual definition when reviewing. Would you mind adding a default impl returning Status::OK() in both Close() in env.h and file_system.h to address @anand1976's feedback? You can possibly do that in the same PR adding the missing comments. Thanks!